### ISSUE TITLE
something is wrong with ap=0

### ISSUE THREAD

**Issue by May2333** [fundamentalvision/Deformable-DETR#27]
after I trained the code followed the readme without any change for 6 epoch, the AP is close to 0.
anyone knows what's wrong with it? and how to solve it?
any help will appreciate!
![image](https://user-images.githubusercontent.com/23239452/103253033-67412780-49ba-11eb-95b0-2722f76e91d8.png)

**Comment by jackroos** [comment]
Could you provide the full command you ran?
BTW, you could check if your setup is correct by evaluating our pre-trained model.

**Comment by lalagood** [comment]
hi, but i found that the pre-trained model could not be downloaded these days

**Comment by lalagood** [comment]
> Could you provide the full command you ran?
> BTW, you could check if your setup is correct by evaluating our pre-trained model.

hi, but i found that the pre-trained model could not be downloaded these days

**Comment by MaLiXiaoGuo** [comment]
我的也是艾

**Comment by zhangjing9701** [comment]
@May2333 
@jackroos 
@MaLiXiaoGuo 

Hi. I met the same problem.  Have you solved it? Thanks very much!!!

**Comment by MaLiXiaoGuo** [comment]
> @May2333
> @jackroos
> @MaLiXiaoGuo
> 
> Hi. I met the same problem. Have you solved it? Thanks very much!!!
I adjested training paraments of "Transformer", and trained more epochs.
But the ap is still very low.

**Comment by May2333** [comment]
I don't know.. So I just fine-tune on official premodel, it seems no problem

**Comment by ghost** [comment]
Hi @May2333, Can you give me your reference source code for your fine-tune. I tried a lot but still can't work.
Thank you in advance.

**Comment by 102424lmy** [comment]
Have you found the reason? I have the same problem as you：No matter how many epoches trained, AP is close to 0. I would appreciate it if you could give me some guidance.

**Comment by HanWangSJTU** [comment]
There is one possible reason: batchsize is too small, but the learning rate remains unchanged. So the network is hard to reach convergence.

**Comment by ishidadadada** [comment]
I have the same problem as you：No matter how many epoches trained, AP is close to 0

However, Test is good result of detecting small obstacle.
Thank you for your help.

**Comment by ishidadadada** [comment]
![n100 loss](https://user-images.githubusercontent.com/80731879/150307132-1548e544-02df-410d-bb4d-53707a3d86d6.png)

私は、道路上の小さな物体のクラスについて識別実験をしています。　バウンディングボックスを安全に囲むことができるにもかかわらず、APが0になってしまうのはなぜでしょうか？

**Comment by ishidadadada** [comment]
I am doing an identification experiment for a class of small objects on the road.　Why is it that the AP is 0 even though the bounding box can be enclosed safely?

**Comment by ishidadadada** [comment]
![2022-01-19 (1)](https://user-images.githubusercontent.com/80731879/150307717-2982ef1f-a876-4b95-841e-9ac38a0530bb.png)

**Comment by ishidadadada** [comment]
I have two hypotheses for the cause of the 0 AP.

1) The custom data set is not available in coco format.
(of course, it has been converted to JSON format)
(2) The official coco-pre trained model is not being used properly.
